### PR TITLE
[WIP] Issue #80: Add fix-dependencies command

### PR DESCRIPTION
**Pull request by rugabarbo**
#80

### CODE REVIEW

**Comment by rugabarbo** [comment]
I plan to polish some things and write tests. But in general, the architecture is ready, and the command even works - you can try it.

**Comment by samdark** [comment]
I'll merge it. Let's have enhancements separately. Thank you, it will save time for sure! 👍

File: src/Component/CodeUsage/NamespaceUsageFinder.php [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+    private function getExistingPHPFilePaths(string $environment): array
+    {
+        $files = [];
+
+        foreach ($this->target[$environment] as $absolutePath) {
+            if (!file_exists($absolutePath)) {
+                continue;
+            }
+
+            // TODO: Check extension
```
Comment (samdark) [inline src/Component/CodeUsage/NamespaceUsageFinder.php:75]: Is that what was going to be improved?

File: src/Component/Composer/ComposerPackage.php [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+    {
+        if ($this->config === null) {
+            $this->config = ComposerConfig::createByFilePath($this->getComposerConfigPath());
+        }
+
+        return $this->config;
+    }
+
+    /**
+     * @param string $specificVendor
```
Comment (samdark) [inline src/Component/Composer/ComposerPackage.php:49]:
```suggestion
     * @param string|null $specificVendor
```

File: src/Component/CodeUsage/NamespaceUsageFinder.php (hunk 1 above)
Comment (rugabarbo) [reply src/Component/CodeUsage/NamespaceUsageFinder.php:75]: No, this is a minor issue.
See #84 :)

File: src/Component/Composer/ComposerPackage.php (hunk 2 above)
Comment (rugabarbo) [reply src/Component/Composer/ComposerPackage.php:49]: Applied in #84 
Thanks :)

**Pull request merged**